lib/kubeclient/common.rb

```diff
@@ -109,6 +109,10 @@ def self.define_entity_methods(entity_types)
         define_method("update_#{entity_name}") do |entity_config|
           update_entity(entity_type, entity_config)
         end
+
+        define_method("patch_#{entity_name}") do |name, namespace = nil, patch|
```

Comment:
It sounds to me that according to this flow, name and namespace are both part of the "what to patch". This order is also consistent with the other methods (Get and Delete), where namespace comes right after name. Changing it might be confusing for the user. @simon3z  what do you think?